src/screens/Forum/ForumHomeScreen.js

```diff
@@ -14,9 +14,18 @@ export default class ForumHomeScreen extends React.Component {
     super(props);
     this.state = {
       forumPosts: [],
+      postLimit: 10,
```

Comment:
I would recommend using react-native hooks instead of state here (https://reactjs.org/docs/hooks-intro.html) in order to simplify state management, if it's possible.